controller.js

```diff
@@ -0,0 +1,50 @@
+var calc = new Calculator()
+window.onload = document
+  .getElementById( 'calculator-output' )
+  .innerHTML = calc.getDisplay()
```

Comment:
I'm not familiar with onload. If I understand the documentation correctly, you're telling your program to call these functions when the `onload` event happens?